WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -43,4 +31,22 @@ extension Color {
     static var wooAmberShade80: Color {
         Color(red: 123.0 / 255.0, green: 7.0 / 255.0, blue: 0.0 / 255.0)
     }
+
+    /// Colors from hi-fi m: p91TBi-bot-p2
+    ///
+    static var posPrimaryTexti3: Color {
```

Comment:
Happy to iterate on the names of the colors, I don't think these are final so hence the i3 append for the moment, since I've also found tricky to remember which one was primary/secondary/tertiary previously.